src/apps/pulse/components/Search.tsx

```
@@ -0,0 +1,163 @@
+            <div style={{marginRight: 10}}><TailSpin color="#FFFFFF" height={20} width={20} /></div> :
+            <Close onClose={handleClose}/> :
+            <Close onClose={handleClose}/>
+          }
+        </div>
+
+        {/* Trending, Fresh, TopGainers, MyHoldings */}
+        <div className="flex">
+          {["🔥 Trending", "🌱 Fresh", "🚀 Top Gainers", "💰My Holdings"].map(item => {
+            return (
+              <div className="flex" style={{backgroundColor: "black", marginLeft: 10, width: 95, height: 30, borderRadius: 10}}>
+                <button className="flex-1 items-center justify-center" style={{backgroundColor: "#121116", borderRadius: 10, margin: 2, color: "grey"}}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)